pkg/raft/tracker/electiontracker.go

```diff
@@ -16,21 +16,24 @@ type ElectionTracker struct {
 	config *quorum.Config
 
 	votes map[pb.PeerID]bool
+	hints map[pb.PeerID]pb.Entry
```

Comment:
I would structure this as:

```golang
type vote struct {
	acc  bool    // vote accepted if true, rejected if false
	last entryID // last entry ID in the voter's log
}

votes map[pb.PeerID]vote
```

So that we don't have to allocate yet another map.